nautobot_ssot/tests/ipfabric/test_nbutils.py

```diff
@@ -23,7 +24,7 @@ class TestNautobotUtils(TestCase):
 
     def setUp(self):
         """Setup."""
-        reg_loctype = LocationType.objects.update_or_create(name="Region")[0]
+        reg_loctype = LocationType.objects.update_or_create(name="Site")[0]
```

Comment:
It we're changing this to be a Site LocationType we should probably be changing the variable name accordingly.